Tasks/ANT/anttask.ts

```diff
@@ -1,15 +1,18 @@
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
 
 import tl = require('vsts-task-lib/task');
+import path = require('path');
```

Comment:
Get all user inputs in the top and assign to variables. Will keep code clean n organized.